apps/testrunner/test_suites/selenium_tests/selenium_test.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import logging
+from selenium import webdriver
+from selenium.webdriver import DesiredCapabilities
+from selenium.webdriver.support.wait import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.common.by import By
+from testrunner.test_suites.selenium_tests.selenium_timer import SeleniumTimer
+
+logger = logging.getLogger(__name__)
+
+class SeleniumTest(object):
+    def get_driver(self):
+        caps = DesiredCapabilities.CHROME
+        #caps['loggingPrefs'] = {'performance': 'ALL'}
+        driver = webdriver.Chrome(executable_path='/usr/lib/chromium-browser/chromedriver',
+                                  service_args=["--verbose", "--log-path=chromelog.log"],
+                                  desired_capabilities=caps)
+
+        return driver
+
+    def get_and_measure(self, url, driver, timer):
+
+        driver.get(url)
+        self.wait_for_page_load(driver)
+        timer.mark_measurement_point(url)
+
+    def wait_for_page_load(self, driver):
+
+        WebDriverWait(driver, 30).until(
+            EC.presence_of_element_located((By.XPATH, '//iframe[@title="VisualDNA Analytics"]')))
+
+    def run(self):
+        result = None
+        try:
+            driver = self.get_driver()
```

Comment:
This looks like a nice place to use Python's context manager:
https://docs.python.org/2/reference/datamodel.html#context-managers